docs/sparkr.md

```diff
@@ -295,8 +294,7 @@ head(collect(df1))
 
 ##### dapplyCollect
 Like `dapply`, apply a function to each partition of a `SparkDataFrame` and collect the result back. The output of function
-should be a `data.frame`. But, Schema is not required to be passed. Note that `dapplyCollect` only can be used if the
-output of UDF run on all the partitions can fit in driver memory.
+should be a `data.frame`. But, Schema is not required to be passed. Note that `dapplyCollect` can fail if the output of UDF run on all the partition cannot be pulled to the driver and fit in driver memory.
 <div data-lang="r"  markdown="1">
```

Comment:
I think we need a new line before the `<div>` ? Right now the `div` markings show up in the generated doc. I've attached a screenshot

![screenshot 2016-07-15 14 11 39](https://cloud.githubusercontent.com/assets/143893/16888609/1d4409fe-4a96-11e6-97db-6ebf05a03774.png)